README.md

```diff
@@ -6,44 +6,50 @@ See more details at https://blocktogether.org.
-website don't matter; You'll only be using this for testing. However, it is
-important that you don't leave the 'Callback URL' blank or you won't be able to
-log in. Fill in any arbitrary URL here - the app will override it at login time.
-After you've created the app, make sure it is set to have read/write permission.
-The write permission is necessary to apply blocks, unblocks, and mutes. You may
-need to add a phone number to your account in order to get read/write
-permission. After you've set read-write permission, the consumerKey and
-consumerSecret listed on apps.twitter.com will be different. Copy the new
-consumerKey and consumerSecret into /etc/blocktogether/config.json in the
-appropriate fields.
-
-You're now ready to start the web frontend:
-
-    bash ./run.sh
-    firefox http://localhost:3000/
-
-Log in for the first time using the 'Log in' button, not the 'Sign up' button.
+First, make sure that you have [Vagrant](https://www.vagrantup.com/) installed.
+Then, from the blocktogether directory, run:
+
+    vagrant up
+
+Second, create an app on Twitter for your local version of blocktogether:
+
+  1. Head to https://apps.twitter.com/ and click "Create New App"
+  2. Fill out form & click "Create your Twitter application". (The description and
+     website don't matter; You'll only be using this for testing. However, it is
+     important that you don't leave the 'Callback URL' blank or you won't be able to
+     log in. Fill in any arbitrary URL here - the app will override it at login time.
+
+  3. Under "Application Settings" > "Access level", click "modify app permissions"
+     and select "Read and Write" access. The write permission is necessary to apply
+     blocks, unblocks, and mutes. (You may need to add a phone number to your
+     account in order to get read/write permission.)
+
+  4. After you've set the read-write permissions, click the "Keys and Access Tokens"
+     tab. (Note that changing your app's permissions will regenerate these keys.)
+
+
+Now, we'll want to save our Twitter credentials to the configfile so BlockTogether
+can access Twitter's API:
+
+  1. From the `blocktogether` directory, run `vagrant ssh`
+  2. Open the config file for editing: `sudo vim /etc/blocktogether/config.json`
+     or with your editor of choice (though you might have to apt-get it)
+  3. Replace the placeholder `consumerKey` and `consumerSecret` fields with your
+     your new credentials (found under your dev app on https://apps.twitter.com)
+
+You're now ready to run BlockTogether locally. :D Start the by first SSH'ing
+into the Vagrant box. From the root `blocktogether` directory:
+
+    vagrant ssh
+
+And then from the Vagrant box:
+
+    cd /vagrant && ./run.sh
+
+You should now be able to access your local version of blocktogether in a browser
+at http://localhost:3000.
+
+Log in for the first time using the 'Log in' button (nope, not the 'Sign up' button).
```

Comment:
Yeah, I remember now: if you 'sign up' with the follow box checked  but there is no userToFollow yet, you'll get an error.